x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/cache/common/CacheFileTests.java

```diff
@@ -380,6 +389,54 @@ public void testFSyncFailure() throws Exception {
         }
     }
 
+    public void testCacheFileCreatedAsSparseFile() throws Exception {
+        assumeTrue("This test uses a native method implemented only for Windows", Constants.WINDOWS);
+        final long ONE_MB = 1 << 20;
```

Comment:
Nit: I think it should be a constant in the upper case or a normal variable in the snakeCase.

Reply:
I pushed c00295fea4e92a5ecddf672f60d34d6a3ff3fa16